contrib/svc-cat-apiserver-aggregation-tls-setup.sh

```diff
@@ -14,9 +14,10 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
-export HELM_NAME=catalog
-export SVCCAT_NAMESPACE=catalog
-export SVCCAT_SERVICE_NAME=${HELM_NAME}-${SVCCAT_NAMESPACE}-apiserver
+export HELM_RELEASE_NAME=${HELM_RELEASE_NAME:-catalog}
+export CHART_NAME=catalog
+export SVCCAT_NAMESPACE=${SVCCAT_NAMESPACE:-catalog}
+export SVCCAT_SERVICE_NAME=${HELM_RELEASE_NAME}-${CHART_NAME}-apiserver
```

Comment:
I'm trying to match https://github.com/kubernetes-incubator/service-catalog/blob/master/charts/catalog/templates/apiserver-service.yaml#L4 which uses https://github.com/kubernetes-incubator/service-catalog/blob/master/charts/catalog/templates/_helpers.tpl#L8